configuration/init/systemd/tedge-dm-agent.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=tedge-dm-agent collects device monitoring measurements and converts them to connectivity provider format.
+After=syslog.target network.target mosquitto.service
+
+[Service]
+User=tedge-dm-agent
+ExecStart=/usr/bin/tedge_dm_agent
```

Comment:
This is okay for that PR, but we will have to consider soon something along the line `/usr/bin/tedge_mapper dm-agent` where the `tedge-mapper` executable is shared by all the mappers.